src/pages/[tab].astro

```diff
@@ -0,0 +1,25 @@
+---
+import type { GetStaticPaths } from "astro"
+import Layout from "../layouts/Base.astro"
+import { getExamples, type Example } from "../utils/index.js"
+
+export type Props = {
+	examples: Map<string, Example[]>
+}
+
+export const prerender = true
+
+export const getStaticPaths: GetStaticPaths = async () => {
+	const examples = await getExamples()
+	return [...examples.entries()].map(([tab]) => ({
+		params: { tab },
```

Comment:
It might be worth slugifying the tab names so they're always lower case and avoid whitespace characters.  I don't trust browsers or OSs when it comes to handling URLs in a sane way

Note: this would need to be updated to use the same slugify method in `Base.astro` so the `<nav>` uses the slugs too, or I guess the slug could be calculated in `utils` 🤔